pkg/webhook/server.go

```diff
@@ -76,6 +76,8 @@ type Server struct {
 
 	// defaultingOnce ensures that the default fields are only ever set once.
 	defaultingOnce sync.Once
+
+	Started bool
```

Comment:
Let's document this field, I'm wondering though if we should expose it as a method rather than a field on the struct, given that this field isn't safe to use concurrently

Reply:
yeah, let's put that behind a method so that it's safe to call concurrently.  See also #1155